src/d3-ext/profile.js

```diff
@@ -58,13 +62,13 @@ ngeo.profile = function() {
    * @type {function(Object)}
    * Hover callback function.
    */
-  var hoverCallback = goog.nullFunction;
+  var hoverCallback = options.hoverCallback || goog.nullFunction;
```

Comment:
Consistency should be second after readability. There is no point in having a code base consistently hard to read.
Moreover, it is just plain and widely used javascript every developer should be familiar with.